src/components/safe-area-layout.component.tsx

```diff
@@ -27,37 +25,40 @@ type Inset = 'top' | 'bottom';
 export interface SafeAreaLayoutProps extends ViewProps, StyledComponentProps {
   insets?: Inset;
   children?: React.ReactNode;
+  backgroundColor?: string;
```

Comment:
This prop is not for managing standard css style property, it takes a string with mapped theme color, so that's why we can't actually set it just in styles, because it wouldn't work like that. That interface ViewProps valid only for rn StyleSheet, but we're going to use themed color there, so I decided to use pass it like a string and get access to the background-color property directly from safeArea component as a string.
<img width="704" alt="Screenshot 2021-01-12 at 18 41 29" src="https://user-images.githubusercontent.com/36364993/104337817-ef6d1400-5506-11eb-8e63-0e24545ea456.png">
<img width="926" alt="Screenshot 2021-01-12 at 18 47 47" src="https://user-images.githubusercontent.com/36364993/104337832-f2680480-5506-11eb-9da4-039092856aec.png">